apps/publisher/themes/default/js/popover.js

```diff
@@ -43,6 +43,33 @@ var popoverFunction = (function(){
     return returnObj;
 })();
 
+var currentPageFunction = (function(){
+    var returnObj = {};
+    returnObj.init = function(){
+        console.log("tmkasun debug yhehaa "+store.publisher.currentPage);
```

Comment:
Thanks, This has been fixed with [this commit](https://github.com/wso2/carbon-store/commit/63493eef6a6b9acc6f9c1381a39e2434a81a90c6)